lib/cc/yaml/nodes/engine.rb

```diff
@@ -4,7 +4,28 @@ module Nodes
       class Engine < Mapping
         map :enabled, to: Scalar[:bool], required: true
         map :checks, to: Checks
-        map :config, to: EngineConfig
+        map :config, to: Mapping
+
+        attr_reader :config
+
+        def visit_key_value(visitor, key, value)
+          if key == "config"
+            raw_value = value.to_ruby
```

Comment:
How is #to_ruby implemented? I'm interested in potential security issues through object injection, which is a key thing this library needs to avoid.